tests/Shared/WorkloadTesting/BuildEnvironment.cs

```diff
@@ -133,6 +133,11 @@ public BuildEnvironment(bool useSystemDotNet = true, TestTargetFramework targetF
         // Set DEBUG_SESSION_PORT='' to avoid the app from the tests connecting
         // to the IDE
         EnvVars["DEBUG_SESSION_PORT"] = "";
+        // Avoid using the msbuild terminal logger, so the output can be read
+        // in the tests
+        EnvVars["_MSBUILDTLENABLED"] = "0";
```

Comment:
1. Why is this start with an underscore?
2. Do we need to set both this and VsTest? I thought VS test gets turned off when msbuild is turned off.

Reply:
> 1. Why is this start with an underscore?

I'm not sure. Maybe the msbuild team intends this to be a private, and temporary knob.

> 2. Do we need to set both this and VsTest? I thought VS test gets turned off when msbuild is turned off.

Looks like this might fixed in newer SDKs - https://github.com/microsoft/vstest/blob/4f16fc041b85e102c7ee783ed4124d733d067c74/src/Microsoft.TestPlatform.Build/Microsoft.TestPlatform.targets#L32-L33

This is what I have in `9.0.100-preview.7.24407.12`:

```xml
  <Target Name="VSTest" DependsOnTargets="ShowInfoMessageIfProjectHasNoIsTestProjectProperty">
    <!-- Unloggable colorized output (cf. https://github.com/microsoft/vstest/issues/680) -->
    <CallTarget Targets="_VSTestConsole" Condition="!$(VsTestUseMSBuildOutput) OR $(MSBUILDENSURESTDOUTFORTASKPROCESSES) == '1'" />
    <!-- Proper MSBuild integration, but no custom colorization -->
    <CallTarget Targets="_VSTestMSBuild" Condition="$(VsTestUseMSBuildOutput)" />
  </Target>
```

Reply:
> Maybe the msbuild team intends this to be a private, and temporary knob.

Yes and it's also an _output_ of MSBuild not an input, so I don't think I understand what's happening here.